exercises/concept/top-secret/.docs/instructions.md

```diff
@@ -0,0 +1,73 @@
+# Instructions
+
+You're part of a task force fighting against corporate espionage. You have a secret informer at Shady Company X, which you suspect of stealing secrets from its competitors.
```

Comment:
I wanted to avoid any implicit political associations so I decided to mention explicitly that this is about corporate espionage. And to avoid suggesting that the student is supposed to be a criminal here 😁 so I went with _fighting_ espionage. I hope this still makes sense.